src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not typical for entity classes in Hibernate. Entities should be unique and managed by the persistence context, so cloning them can lead to unexpected behavior. Consider removing the clone method and the Cloneable interface unless there is a specific requirement for it.